.coveragerc

```diff
@@ -1536,6 +1536,7 @@ omit =
     homeassistant/components/zwave_me/switch.py
     homeassistant/components/electrasmart/climate.py
     homeassistant/components/electrasmart/__init__.py
+    homeassistant/components/refoss/*
```

Comment:
You cannot remove all files from being tested. The config flow needs to be tested at least.